src/App.js

```diff
@@ -1,34 +1,29 @@
-import React from 'react';
-import { Switch, Route } from 'react-router-dom';
+import React, {useContext} from 'react';
+import {Routes, Route, Navigate} from 'react-router-dom';
 import NavBar from './components/NavBar';
 import Profile from './pages/Profile';
 import Home from './pages/Home';
 import SignIn from './pages/SignIn';
 import SignUp from './pages/SignUp';
 import './App.css';
+import {AuthContext} from "./Context/AuthContext";
 
 function App() {
-  return (
-    <>
-      <NavBar />
-      <div className="content">
-        <Switch>
```

Comment:
Werk jij nog met een oudere versie of kan de switch ook nog steeds in de nieuwe versie? 
Ik heb zonder switch en exact path:
 return (
    <>
      <NavBar />
      <div className="content">
          <Routes>
              <Route path="/" element={<Home/>}/>
              <Route path="/profile" element={isAuth ? <Profile/> : <Navigate to="/"/>}/>
              <Route exact path="/register" element={<Register/>}/>
              <Route exact path="/signin" element={<SignIn/>}/>
          </Routes>
      </div>
    </>
  );
}